ci/test/interrupt

```diff
@@ -0,0 +1,58 @@
+#!/bin/bash
+set -ex
+
+source ./common
+init_test_repo
+cd $test_repo_path
+
+git clone https://github.com/cplee/github-actions-demo.git
+
+cd github-actions-demo
+export PHONY_SECRET=foo
+popper run & pid=$!
+#Give popper run some time to create dockers
+sleep 5
+
+# -2 is for SIGINT or Ctrl-C
+kill -2 $pid
+
+#Give popper run some time to remove dockers and then check
+sleep 5
+check="deploy branch-filter test lint install"
+actual=`docker ps -a --format "{{.Names}}"`
+for a in $actual
+do 
+  for b in $check
+  do 
+    if [ $a = $b ] 
+    then echo "fail" >> fail
+    fi
+  done
+done
+
+counter=`cat fail|wc -l`
+
+#A test for --parallel run
+popper run & pid=$!
```

Comment:
this has to make use of the `--parallel` flag. In addition, the workflow being executed does not contain multiple actions in a single stage, so running with the `--parallel` flag will end up being the same as running without it. We can reuse the workflow defined in the [test for the parallel](https://github.com/systemslab/popper/blob/master/ci/test/parallel_stage_exec) feature here, so that we ensure multiple actions run on the same stage. We might need to modify the actions so that they run for longer (e.g. instead of running `ls`, we can run `sleep 100` or something alike).

Reply:
I think I was using `--parallel` option. But I might have overwritten it somehow, hence the miss. Will change the workflow and update the PR